### DISCUSSION TITLE
Deploying for production with current beta. Good idea?

### DISCUSSION THREAD

**Discussion by Juleszz** [saleor/saleor#8468] [Q&A]
We've been developing an ecommerce site with Saleor a while now and it looks like we're about to launch.

Is there anything we should be aware of releasing the site using the current 3.1 beta? We're not using all features but those we do were tested very intense without any errors.

Any thoughts on this would be appreciated! 🙌

**Comment by maarcingebala** [comment]
3.1 (which mirrors the `main` branch) is currently in alfa while 3.0 (branch `3.0`) is beta. 3.1 contains some new features that are unavailable in 3.0, but they are still being tested and may be subject to change if we find some bugs or inconsistencies in the API schema.

**Comment by maarcingebala** [comment]
3.0 is already used in production in Saleor Cloud and we usually recommend going with this version.
